utils/kata-manager.sh

```diff
@@ -408,6 +408,11 @@ configure_containerd()
 	local enable_debug="${1:-}"
 	[ -z "$enable_debug" ] && die "no enable debug value"
 
+	local http_proxy_set="${2:-}"
```

Comment:
Can reduce `http_proxy_set` and `http_proxy` into one? IMO a non-empty `http_proxy` equals `http_proxy_set=true`

Reply:
Agreed. Pushed the changes.